src/store/state/state-model.ts

```diff
@@ -16,6 +16,8 @@ export const stateModel: StateModelIF = {
     certifiedBy: ''
   },
   currentStep: 1,
+  minStep: 1,
+  maxStep: 3,
```

Comment:
Isn't minStep always 1?

Couldn't maxStep be computed?

I'm thinking there has to be a clean way to implement this without additional state properties.

Reply:
Sure I can take it from that state and make it a local or computed property

Reply:
Have made the changes to make this a computed property